src/impl/core/dappletContext.ts

```diff
@@ -12,23 +13,19 @@ import { GithubDappletProvider } from '../providers/githubDappletProvider';
 // instantated once on Wallet start
 export class DappletContext {
     private _dappletProvider: DappletProvider;
+    private _featuresRegistry: FeaturesRegistry;
```

Comment:
wording "featureRegistry" looks more convenient. Check google result for "FeaturesRegistry" vs "FeatureRegistry".
The term "registry" already implies plural.